qlib/config.py

```
@@ -23,17 +23,21 @@
 from typing import TYPE_CHECKING
 
 from qlib.constant import REG_CN, REG_US, REG_TW
+from qlib.utils import hash_args, get_module_logger
```

Ruff (0.11.9)

`qlib.utils.hash_args` imported but unused

Remove unused import

(F401)